phoneApp/js/app.js

```diff
@@ -0,0 +1,183 @@
+				        {
+				        name: 'Елена',
+				        lastName: 'Лещенко',
+				        email: 'ElenaLeshenko@ec.ua'
+				      },
+				        {
+				        name: 'Ольга',
+				        lastName: 'Новикова',
+				        email: 'OlgaNovikova@ec.ua'
+				      },
+				        {
+				        name: 'Наталья',
+				        lastName: 'Шемякина',
+				        email: 'ShemyakinaN@ec.ua'
+				      },
+				        {
+				        name: 'Анна',
+				        lastName: 'Донцова',
+				        email: 'AnnaDontsova@ec.ua'
+				      },
+				        {
+				        name: 'Влад',
+				        lastName: 'Яма',
+				        email: 'VladYama@ec.ua'
+				      },
+				        {
+				        name: 'Кира',
+				        lastName: 'Воробьева',
+				        email: 'Kira1990@ec.ua'
+				      },
+				        {
+				        name: 'Виктор',
+				        lastName: 'Кривенко',
+				        email: 'ViktorKriv@ec.ua'
+				      }
+				    ],
+			activePage: 'contacts'
+		};
+
+        this.pages = {
+            contacts: new ContactsPage(this.state),	// тут передали ссылку на this.state
+			adduser: new AddUser(this.state),
+            keypad: new KeypadPage(this.state),
+            editcontact: new EditContact(this.state),
+            user: new User(this.state)
+        };
+
+        this.initializeRouter();
+        this.switchRouter();
+
+    }
+
+    initializeRouter () {
+        const mountNode = document.getElementById('mountNode');
+        //console.log(mountNode);
+        mountNode.innerHTML = `
+        <div id="app"></div>
+        <footer class="footer">
+            <div class="container bottom-radius">
+                <nav class="main-nav">
+		            ${this.renderLink({href:'contacts', glyphicon:'search', text:'Contacts', active: true})}
+		            ${this.renderLink({href:'keypad', glyphicon:'th', text:'Keypad', active: false})}
+		            ${this.renderLink({href:'edit-contact', glyphicon:'pencil', text:'Edit contact', active: false})}
+		            ${this.renderLink({href:'user', glyphicon:'user', text:'User', active: false})}
+		            ${this.renderLink({href:'add-user', glyphicon:'plus', text:'Add user', active: false})}    
+                 </nav>
+             </div>
+         </footer>`;
+
+         //this.initializeRouterHandlers();
+         this.appDOMNode = mountNode.querySelector('#app');	// сюда будем делать рендер всех страниц 
+		   													// и это не будет затрагивать футер и его события
+    }
+
+	renderNewPage() {
+
+		this.appDOMNode.innerHTML = this.pages[this.state.activePage].render();
+	}
+
+	renderLink(options) {
+		let {href, glyphicon, text, active} = options;
+		let activeClass = active ? 'active' : '';
+
+		return `<a href="${href}" class="tab ${activeClass}">
+		          <span class="glyphicon glyphicon-${glyphicon}" aria-hidden="true"></span>
+		          <span class = "tab-text">${text}</span>
+		        </a> `
+	}
+
+
+    switchRouter() {
+    	const parent = document.querySelector('.main-nav');
```

Comment:
I think we can make such selector once - and remember the link to it, no need to make it on every